kimchi/src/prover_index.rs

```diff
@@ -19,14 +19,13 @@ type Fr<G> = <G as AffineCurve>::ScalarField;
 type Fq<G> = <G as AffineCurve>::BaseField;
 
 //~
-//~ ### The prover Index
```

Comment:
can you actually revert that? This is what is causing the test issue (as these comments `//~` are special comments that generate the spec

Reply:
Oh thats interesting, good to know those type of comments actually have a usage!

Reply:
yeah, they get extract to create this specification here which I'm working on at the moment: https://o1-labs.github.io/proof-systems/specs/kimchi.html